events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/ErrorResponse.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package com.bounswe.bounswe2017group3;
+
+import java.io.Serializable;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ *
+ * @author gokce
+ */
+
+@Table(name = "errors")
+@Entity
```

Comment:
`@Entity` and `@Table` are for the objects in our database. Since this error class is not persisted in our database, these are probably not necessary 😄